pkg/server/commands/listusers/list_users_rpc_test.go

```diff
@@ -1264,7 +1290,10 @@ func (testCases ListUsersTests) runListUsersTestCases(t *testing.T) {
 			}
 
 			typesys, err := typesystem.NewAndValidate(context.Background(), model)
-			require.NoError(t, err)
+			if err != nil {
+				t.Log("warn! model is invalid")
```

Comment:
What's the rationale here? It feels odd to have expectations of system behavior on invalid models. If the model is invalid and we're not detecting, then that's the responsibility of other code. It shouldn't be this test and the component it is testing's responsibility to ensure correct behavior given invalid constraints.

Reply:
AFAICT, this is to run tests against old models that exhibited cycles, prior to the appropriate model validation. My assumption is that `typesystem.NewAndValidate` should enforce these new rules but we still need to be able to test older models too.

Reply:
1.0 models are no longer. We wouldn't evaluate Check or ListUsers on old models as it is no longer a supported schema version.

Reply:
Apologies for the confusion. I updated the PR with a comment explaining why I did this. Let me know what you think 🙇‍♀️

Reply:
As per offline discussion: removed test for the model with no entrypoints.